lms/djangoapps/discussion/views.py

```diff
@@ -485,6 +486,7 @@ def _create_discussion_board_context(request, course_key, discussion_id=None, th
         'upgrade_link': check_and_get_upgrade_link(request, user, course.id),
         'upgrade_price': get_cosmetic_verified_display_price(course),
         # ENDTODO
+        'retired_username': RETIRED_USERNAME,
```

Comment:
@viadanna, since this is just some text, any user with that name will get this functionality :wink:

Reply:
@lgp171188 Yeah, that's the `Deleted user` username [used](https://github.com/edx-solutions/edx-platform/pull/1403/files/b1142e31fbb81649609893fbc1158dc03d779033#diff-df4681b6dbe6bbc209ddf0f521adb0a6R45) for retired users. Since we don't store any information on retired users and this username is always used when retiring, that's how I could identify retired users.

Another matter is backwards compatibility. If we were to add a field to the user stating it was retired, it wouldn't work with users retired in the past. But if the retired username changes in the future, it won't work as well.